README.md

```diff
@@ -1,23 +1,23 @@
 # Č
 
-The czech library for C. Why? Because we could, of course!
+The Czech library for C. Why? Because we could, of course!
 
 Please contribute your other ~~horrible~~ awesome and creative translations
 so that we can write a fully czech code to suit all our needs.
 
 ## Using
 
 This project resides solely in a single header file, aptly named `czech.h`.
-Download that file alone, or along this whole repository, and then, after
-including it in your source files like you would with any other header, you
-will be fully set up! (Just make sure you use relatively modern compiler)
+Download that file alone, or the entire repository, and then, after
+including it in your source files like you would any other header, you'll
+be good to go! (Just make sure to use a relatively modern compiler)
 
-We have this project tested and provably working with
```

Comment:
I think this line was fine as was but I'm alright with changing it 🤷  tho I prefer "works with (a compiler)" over "works for (a compiler)"